webpack.config.js

```diff
@@ -66,6 +67,23 @@ if (process.env.TESTING_MODE === 'true') {
 }
 */
 
+const babelLoaderExcludes = [
+  /\.test\.js$/,
+  /fixtures\.js$/,
+  /core-js/,
+];
+// Inlucde any modules that need to be transpiled by babel-loader
+const transpileDependencies = [
+  'react-visibility-sensor',
+];
+if (devMode) {
+  // Don't transpile any dependencies in /node_modules except those found
+  // in transpileDependencies array
+  babelLoaderExcludes.push(
+    new RegExp(`node_modules/(?!(${transpileDependencies.join('|')})/).*`),
```

Comment:
```suggestion
    new RegExp(`node_modules(?!(/|\\\\)(${transpileDependencies.join('|')})).*`),
```
The `$RefreshSig$ is not defined` error is related to Windows vs Linux module path issues. This change makes the error go away for me, the app to build in watch mode, and the fast refresh to work based on my testing of a few changes in random React components.

Reply:
Nice!  Good find